### ISSUE TITLE
[v2.0.4] Values not delimited when format string is empty

### ISSUE THREAD

**Issue by sanmai-NL** [slog-rs/slog#133]
In slog v2.0.4:

```Rust
let logger = &slog::Logger::root_typed(drain.fuse(), 
    o!("CARGO_PKG_NAME" => env!("CARGO_PKG_NAME"), 
       "CARGO_PKG_VERSION" => env!("CARGO_PKG_VERSION")));
// ...
slog_debug!(logger, ""; "address" => address);
```

Actual:
```
May 08 15:29:37.119 DEBG address: [::1]CARGO_PKG_VERSION: 0.1.0CARGO_PKG_NAME: test
```

Expected something like:
```
May 08 15:37:40.987 DEBG, , address: [::1], CARGO_PKG_VERSION: 0.1.0, CARGO_PKG_NAME: test
```

**Comment by dpc** [comment]
This is most probably a `slog-term` bug, where `needs_comma` is not being set after output every KV. Will look into it once I find some time. Feel free to submit a PR yourself.

**Comment by dpc** [comment]
Fixed in `slog-term 2.0.1`
